packages/alfa-rules/test/sia-r41/rule.spec.tsx

```diff
@@ -171,3 +173,44 @@ test(`evaluate() can't tell if two links that have the same name references
     ),
   ]);
 });
+
+test(`toJSON() with minimal verbosity produces target with correct serialization ids`, async (t) => {
```

Comment:
I'm not sure this is relevant for this rule 🤔 Is it a test for serialisation of `Outcome` (we're missing these tests in `alfa-act` 🙈 )? Anyway, doesn't hurt.

Reply:
Yeah it's a bit awkward, but I wanted to have sort of an end-to-end and chose R41 because it show cases that the options are passed through both `Array`, `Outcome` and `Group` before reaching the `Node`.

There should also be tests of the individual units, but I thought having this would be good for now. When I get time, I can add some more tests in the individual packages.